### DISCUSSION TITLE
<title>

### DISCUSSION THREAD

**Discussion by evansso** [community/community#60480] [Other Feature Feedback, Questions, & Ideas]
Can I Start to learn react with some knowledge of Javascript

Product Feedback

### Body

i want to leaarn react nd also javascript.what should i do  for now . at javascript i have learned conditionals scope functions data types variables. what advice can you give me

**Comment by raanausama** [comment]
That's awesome that you're eager to learn React and JavaScript. You're already off to a great start with your understanding of conditionals, scope, functions, data types, and variables in JavaScript.
Here's what you can do next:

1. Solidify Basics: Make sure you have a strong grasp of the fundamentals you've learned. Practice creating different types of functions, using different data types, and working with conditionals. This will build a strong foundation for your journey.
2. HTML & CSS: Before diving into React, get comfortable with HTML and CSS. They are the building blocks of web development and will be essential when working with React components.
3. DOM Manipulation: Learn how to interact with the Document Object Model (DOM) using JavaScript. This is important because React also deals with the DOM. Practice changing elements on a webpage using JavaScript.
4. ES6+ Features: Explore more advanced JavaScript features like arrow functions, template literals, destructuring, and classes. These will come in handy when you start working with React.
5. React Basics: Start with the basics of React. Learn about components, props, and state. You can follow online tutorials or courses to get hands-on experience.
6. Mini Projects: As you learn, work on small projects that use React. This could be a to-do list, a simple calculator, or a weather app. Applying what you learn is key to solidifying your knowledge.
7. Practice Regularly: Consistency is key. Set aside some time each day to learn and practice. Over time, you'll see steady progress.
8. Stay Curious & Patient: Learning takes time, so be patient with yourself. Stay curious, keep exploring, and have fun while you're at it!

Remember, learning is a journey, and you're on the right path...
